client/my-sites/checkout/src/components/item-variation-picker/util.ts

```diff
@@ -14,6 +14,11 @@ export function getItemVariantCompareToPrice(
 		return undefined;
 	}
 
+	// Ignore a 1 month discount when calculating the discount percentage
+	if ( compareTo.introductoryInterval === 1 && compareTo.introductoryTerm === 'month' ) {
+		return compareTo.priceBeforeDiscounts * 2;
+	}
+
```

Comment:
Does the multiplication by 2 only apply if the `variant` has a 2 year term? it seems to have an interesting effect on showing discounts in the plan select drop-down. The monthly plan shows as a 94% discount, which is not correct.

![Screenshot 2023-09-08 at 11 56 35 AM](https://github.com/Automattic/wp-calypso/assets/18016357/fd0df9b6-0475-4da1-92d7-6f1b1d128726)

Reply:
Oh nice catch, I didn't account for that thank you, I'll get that fixed

Reply:
I got a fix and everything seems to be working. As we've always talked about before though it does seem this along with pricing stuff in general could be refactored in a more robust way. But I think this works for now to solve the immediate issue